### PR TITLE
Clarify in spec review that copyright isn't always Eclipse Foundation

### PR DESCRIPTION
**Pull request by scottkurz**
Signed-off-by: Scott Kurz <skurz@us.ibm.com>

Looking at the review checklist, I realized Batch was a special case with an IBM copyright.   

Maybe this rewrite is a bit wordy so feel free to condense.

### CODE REVIEW

**Comment by scottkurz** [comment]
> My understanding is that the final versions of the specs are required by the EFSL to be copyright Eclipse Foundation.

I wasn't referring to the copyright in the spec, but the copyright in the generated Javadoc.   From this [thread](https://www.eclipse.org/lists/jakartaee-spec-project-leads/msg00157.html) I thought we cleared up that the generated Javadoc needed the EFSL license but hadn't seen that the copyright holder should be listed as "Eclipse Foundation".    In any case, will be good to hear clarification.

**Comment by waynebeaton** [comment]
> My understanding is that the final versions of the specs are **required** by the EFSL to be copyright Eclipse Foundation.
> 
> @waynebeaton please confirm.

Confirmed.

**Comment by scottkurz** [comment]
@waynebeaton thx for taking a  look.. did you see though that I was referring to the copyright in the Javadoc, not the copyright in the specs?

**Comment by bshannon** [comment]
The javadoc is *part of* the spec.

**Comment by kwsutter** [comment]
Thanks for this clarification, @waynebeaton and @bshannon.  I tried asking this question on this morning's call, but we were trying to do so many things, it got lost in the shuffle.  At least now we have an answer.  Thanks!

**Comment by starksm64** [comment]
I don’t see how that is possible for CDI, for example, as the project has just been moved over to Eclipse, but the copyrights have not been assigned to Eclipse. The Apache License does not require reassignment of copyright in order to be able to relicense as EFSL. Our legal team has suggested all that is required is to have a statement like:

This specification is licensed under the Eclipse Foundation Specification License 1.0; this specification is based on material that is licensed under the Apache License, version 2.0.

**Comment by bshannon** [comment]
The published specifications need to be under EFSL.  And yes, they're derived from
content licensed under other licenses.  I'll leave it to Eclipse legal to determine if or
how we need to attribute the license of the content they're derived from.
